packages/core/src/components/forms/radioGroup.tsx

```diff
@@ -89,11 +94,12 @@ export class RadioGroup extends AbstractComponent<IRadioGroupProps, {}> {
     }
 
     private getRadioProps(optionProps: IOptionProps) {
-        const { name } = this.props;
+        const { name, inline } = this.props;
         const { value, disabled } = optionProps;
         return {
             checked: value === this.props.selectedValue,
             disabled: disabled || this.props.disabled,
+            inline: inline == null ? false : inline,
```

Comment:
no need for a ternary when dealing with booleans. should be safe to just pass `inline` directly, or you could assign default prop value.